src/interaction/InteractionManager.js

```diff
@@ -761,7 +767,7 @@ export default class InteractionManager extends EventEmitter
      *  interactionEvent, displayObject and hit will be passed to the function
      * @param {boolean} [hitTest] - this indicates if the objects inside should be hit test against the point
      * @param {boolean} [interactive] - Whether the displayObject is interactive
-     * @return {boolean} returns true if the displayObject hit the point
+     * @return {number} returns 1 or 2 if the displayObject hit the point, 0 if not
```

Comment:
Since it's a private method, I don't think it's too important to do so. For those looking at the code, I feel like the name of the constants should be sufficient.